pwndbg/commands/context.py

```diff
@@ -70,17 +75,75 @@ def validate_context_sections():
 
 class StdOutput(object):
     """A context manager wrapper to give stdout"""
-    def __enter__(*args,**kwargs):
+    def __enter__(self):
         return sys.stdout
-    def __exit__(*args, **kwargs):
+    def __exit__(self, *args, **kwargs):
+        pass
+    def __hash__(self):
+        return hash(sys.stdout)
+    def __eq__(self, other):
+        return type(other) == StdOutput
```

Comment:
```suggestion
        return type(other) is StdOutput
```

Could also be `isinstance(other, StdOutput)` but this would also accept subclasses.